club/coordinators.md

```diff
@@ -128,15 +142,43 @@ img.default {
 </div>
 
 <div style="margin-bottom: 1rem">
-  <h2 style="margin-bottom: 1rem">President</h2>
+  <h2 style="margin-bottom: 1rem">Coordinator</h2>
   <div class="row" style="margin-bottom: 1rem">
     <div class="col-lg-3">
-      <img src="/assets/img/club_photos/Kori.jpeg" alt="Picture of Kori South" class="rounded shadow coordinator">
+      <img src="/assets/img/club_photos/Ari_Lacanienta.jpg" alt="Picture of Ari Lacanienta" class="rounded shadow coordinator"><!--TODO: Ari Pic -->
```

Comment:
This seems out of order. They should be in alphabetical order by last name.

Reply:
Oh okay. I just thought it made sense to have the president at the top.